### PR TITLE
(WIP) feat (iplb) add vrack configuration to iplb

### PR DESCRIPTION
**Pull request by Mboulianne**

### CODE REVIEW

File: client/app/iplb/vrack/iplb-vrack-edit.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
+<!-- <pre><code>{{ $ctrl.model | json }}</code></pre> -->
```
Comment (Mboulianne) [inline client/app/iplb/vrack/iplb-vrack-edit.html]: I'll remove this once everything is done

File: client/app/iplb/vrack/iplb-vrack.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<button data-ng-click="$ctrl._associateVrack()">Associate vrack TEST</button>
+<button data-ng-click="$ctrl._deAssociateVrack()">Deassociate vrack TEST</button>
+
+TEST {{ $ctrl.creationRules.data }}
```
Comment (Mboulianne) [inline client/app/iplb/vrack/iplb-vrack.html]: I'll remove this once everything is done

File: client/app/helper/controller-modal-helper.service.js [hunk 3]
```diff
@@ -71,9 +71,17 @@ class ControllerModalHelper {
...
 
     showDeleteModal (config = {}) {
-        config.submitButtonText = this.$translate.instant("common_delete");
+        config.submitButtonText = config.submitButtonText || this.$translate.instant("common_delete");
         return this.showConfirmationModal(config);
     }
+
+    showVrackDeactivateModal (vrack, config = {
+        titleText: "Désactiver le vRack",
+        text: `Êtes-vous sûr de vouloir désactiver le vRack « ${vrack.displayName} »`,
```
Comment (Netnix) [inline client/app/helper/controller-modal-helper.service.js]: translation?

**Comment by jleveugle** [comment]
@Netnix i close this PR due to inactivity

Feel free to re-open it if needed

**Pull request merged**